_docs/launch-phases.md

```diff
@@ -1,22 +1,22 @@
 ---
```

Comment:
This file is not listed in the left bar (_data/navigation_docs.yml). It is still searchable, so we should remove them.

Could you do a sweep, to remove any file that is not defined in the `_data/navigation_docs.yml`?

Reply:
Fixed